src/components/PeopleAndRoles/OrgPerson.vue

```diff
@@ -19,12 +19,12 @@
                 id="org-person-form"
                 ref="orgPersonForm"
                 v-model="orgPersonFormValid"
-                v-on:submit.prevent="addPerson"
+                v-on:submit.prevent
```

Comment:
Old code didn't make sense (to me). New code works correctly (ie, pressing Enter in form fields does NOT submit the form).